### DISCUSSION TITLE
attribute update

### DISCUSSION THREAD

**Discussion by vprint** [MscrmTools/XrmToolBox#1019] [Q&A]
Hello,

I have a task to execute which consist of taking the part of an attribute A and then to update an attribute B with this part.

Exemple : 
Entities "Town" has 2 attribute : 
       "Name"
       "Town"

If "Name"="Paris (75001)" then we should obtain "Town"="Paris". 
The idea is to remove the 8 last characters of "Name" attribute. I've tried to do that with the Workflow creator but it does not provide tools to do it. 
I also checked the Dynamics365workflowTools but it does not work with MSCRM 2011.

Do you think it's possible to execute this task with XrmToolBox ?

Cheers

Valentin

**Comment by MscrmTools** [comment]
I guess you will need code to perform this. You could write the necessary code with XrmToolBox tool "Code Now".
Or write a small command line program to do the job
